### DISCUSSION TITLE
Feature Request: Flash/Ping Shortcuts

### DISCUSSION THREAD

**Discussion by NessMB** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#17] [Ideas]
This is easily my favourite mod as it makes being the ship manager and a ton more fun (I'm always the ship manager) and makes radar boosters actually good, but I think flashing and pinging could really use some shortcuts because having to type "flash <first 3 letters of radar's name>" every time gets a little tiring and opens a lot of oppurtunity to mistype when flashing in quick succession, especially when trying to manage multiple radar boosters at a time.

I would love if you could make it so if you type "flash" or "f" while looking at a radar booster in the terminal, it would flash the radar booster currently on the monitor and "ping" or "p" would ping it

I would also like if you could add commands to "flash all" to flash all radar boosters, "ping all" to ping all radar boosters, and "teleport all" to teleport all radar boosters back to the ship. (These should probably have configs to turn them off and on because they might be pretty powerful)

**Comment by MrHydralisk** [comment]
I was considering terminal related part for the mod, but kinda haven't figured it out back then and it possible that it will require additional library mod that allow custom terminal commands, so I postprone it. When I will revisit LC mods I will definetly try to see what can be done about terminal.

Currently I had lessen the need for terminal commands like that by using Touchscreen mod, that allow you to just click on RB on screen to trigger it flash or sound.

**Reply by NessMB** [reply]
Thank you for your response. I appreciate you planning to look into adding terminal commands for the radar boosters when you return to modding LC. Your mod is amazing and I look forward to any improvements you may make in the future!

I've been looking for mods to make RB management more convenient and Touchscreen totally helps. I tested it and Touchscreen isn't perfect, but it's certainly helpful! Thank you for suggesting it!

**Reply by ZachHaber** [reply]
@NessMB - If you are still looking for it, I've added `flash` and `ping` commands to this mod: https://thunderstore.io/c/lethal-company/p/ZachHaber/ExtraTerminalCommands/
There's no `flash all` or `ping all`, but I'm not entirely sure how useful those would be.

**Reply by NessMB** [reply]
@ZachHaber I've been using the Beauver version of this mod for a long time now (not knowing it was discontinued). The Flash and Ping commands from your version are definitely helpful so I'll be using this one from now on since it's just a straight improvement. Thanks!

I do have a suggestion though. Would it be possible to add configs to be able to make a list of the commands/shortcuts for every extra command in the terminal? So the user could customize the commands. If they wanted the option to make the "Flash" command to "F" or "FL" or "Ping" to "P" or "PI", for example. 
I suggest this because in the most recent update, you removed a lot of aliases, so maybe someone might miss them. It's a really small thing, but it would be cool and helpful.

Idk how easy or possible that would be though.
No pressure or anything. You've done a great job already! :D

**Reply by ZachHaber** [reply]
I've pushed a new version with configurable aliases :) Enjoy!

**Reply by NessMB** [reply]
@ZachHaber I've meant to thank you (but I kept forgetting and getting distracted every time with Lethal Company), but thank you so much. This is perfect! And I have been enjoying it! 

Thank you so much! :D
